mantidproject/mantid
# Fixes for building against Python 3

Fixes the mantid Python module C++/Python code & MantidPlot C++ Python code to be compatible with both Python 2 & 3.

Note that the Python plugins have not been touched and require more fixes.

**To test:**
- Check that the code base still builds with Python 2 and both Python works through vanilla Python, IPython 2 and MantidPlot.
- Check that the code base now builds with Python 3. As noted above the plugins don't work yet so in order use the mantid module you'll need to do `echo "python.plugins.directories=" >> ~/.mantid/Mantid.user.properties`

No issue number

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
